docs/examples/pre-qualification/multi-stage.csv

```diff
@@ -1,14 +1,14 @@
 First stage,Second stage
-"* The procuring entity issues an invitation to participate in the process
+"* The buyer or procuring entity issues an invitation to participate in the process
 
-* Suppliers submit requests to participate
+* Potential suppliers submit requests to participate
 
-* The procuring entity assess suppliers based on their submissions
+* The buyer or procuring entity assesses the requests
 
-* The procuring entity establishes a list of qualified or selected suppliers","* The procuring entity issues an invitation to bid to the suppliers on the list
+* The buyer or procuring entity establishes a list of qualified or selected potential suppliers","* The buyer or procuring entity issues an invitation to bid to the potential suppliers on the list
 
-* Suppliers submit bids
+* Tenderers submit bids
```

Comment:
"Tenderers" might make things tricky.

Ideally, "tenderers" would have been reserved for organizations that submit a bid. However, `tender.tenderers` is meant to collect the submitters in the first stage, whether it is a bid or a request to participate (see #906 and #909).

Anyhow, if we use "Potential suppliers" here, then it is consistent with the above, and it also means we're not introducing a new term in this file.

```suggestion
* Potential suppliers submit bids
```